projects/GKv3/GKComponents/GKUI/Components/HyperView.cs

```diff
@@ -333,30 +335,55 @@ private void DoPaint(Graphics gfx)
                 } finally {
                     fAcceptFontChange = true;
                     if (brush != null) brush.Dispose();
-                    if (font != null) font.Dispose();

```

Comment:
Fonts have shared handler, so disposing one font breaks fonts with the same family. Do not dispose fonts.

Reply:
👍